implement-cowsay/cow.py

```diff
@@ -0,0 +1,13 @@
+
+import argparse
+import cowsay
```

Comment:
As this is external dependency, it should be declared somewhere - in simple case, it could be `requirements.txt` file.